source/vibe/core/core.d

```diff
@@ -877,18 +892,64 @@ unittest {
 
 
 /**
-	Returns a new armed timer.
-
-	Note that timers can only work if an event loop is running, explicitly or
-	implicitly by running a blocking operation, such as `sleep` or `File.read`.
+	Creates a new timer, that will fire `callback` after `timeout`
+
+    There are two kinds of timer: one-off timers and periodic timers.
```

Comment:
Strictly speaking, the same kind of timer can be dynamically configured for those to two modes (i.e. a timer created as one-shot can later be rearmed as periodic and vice-versa).

Reply:
Indeed, it is more of a semantic classification. I'll try to rephrase.